snowplow_tracker/event_store.py

```diff
@@ -0,0 +1,142 @@
+# """
+#     event_store.py
+
+#     Copyright (c) 2013-2022 Snowplow Analytics Ltd. All rights reserved.
```

Comment:
Btw, we should make an issue to update the year in copyright to 2023.

Reply:
Will do, have a ticket in Jira already!